src/_common/Tab.js

```diff
@@ -21,13 +21,13 @@ export default class Tab extends Component {
     };
 
     render() {
-        const { imgSrc, selected, showIcon, showText, text, onMouseDown, onClose, closable } = this.props;
+        const { imgSrc, selected, showIcon, showText, text, onClick, onClose, closable } = this.props;
 
         return (
             <div
                 role="tab"
                 aria-selected={selected}
-                onMouseDown={onMouseDown}
+                onClick={onClick}
```

Comment:
From my understanding, nothing to do with bubbling, which is already handled in DropDown component.

It's gotta do with how frequent the browser capture a click / mousedown event

When using mousedown, things changes once you press, and before you release (ie. mouseup) the dom changes, and thus detecting mousedown again in another element, then you release, which is a mouseup, so the 2nd mousedown combine with mouseup creates a click event, which mess things up 

I've tried abusing stop propagation and it did not work.